packages/git-utils/src/index.ts

```diff
@@ -21,6 +30,25 @@ export async function getCurrentBranch () {
   }
 }
 
+export async function getCurrentTags () {
+  try {
+    const currentHead = await getCurrentHead()
+    if (!currentHead) {
+      return []
+    }
+
+    const { stdout } = await execa('git', ['show-ref', '--tags', '-d']).pipeStdout(execa(
+      'grep', [currentHead]
+    )).pipeStdout(execa(
+      'sed', ['-e', 's,.* refs/tags/,,', '-e', 's/\^{}//']
```

Comment:
I don't think this will work on Windows.

Reply:
You are right, I didn't think of that.